osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/services/MonitoringService.java

```diff
@@ -82,9 +82,13 @@ public void handleGetPQValuesResponse(final GetPQValuesResponseDto getPQValuesRe
 
         LOGGER.info("handleResponse for MessageType: {}", messageType);
 
+        final String deviceIdentification = ids.getDeviceIdentification();
+        final String organisationIdentification = ids.getOrganisationIdentification();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "organisationIdentification" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1481)